components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/SequenceConfig.java

```diff
@@ -46,6 +49,7 @@
     private String authenticatedIdPs;
 
     private AuthenticatorConfig authenticatedReqPathAuthenticator;
+    private List<String> requestedAcr;
```

Comment:
SequenceConfig looks like config, but it is really a container for runtime data. e.g.
private AuthenticatedUser authenticatedUser;
private String authenticatedIdPs;